dictionaries/gdscript/cspell-tools.config.yaml

```
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/streetsidesoftware/cspell/main/packages/cspell-tools/cspell-tools.config.schema.json
+
+targets:
+  - name: 'gdscript'
+    sources:
+      - filename: 'src/gdscript.txt'
+        maxDepth: 1 # This is set to 1 to prevent initial builds from taking too long.
```

```suggestion
        split: true
        allowedSplitWords:
          - ../en_US/en_US.trie
          - ../software-terms/dict/softwareTerms.txt
```

We can tell the complier to split the words using known words. This will reduce the size of the dictionary.

This is how the `cpp` dictionary does it.